bigquery/cloud-functions/diagnostics/main.py

```diff
@@ -50,10 +52,13 @@ def delete_file(file_name):
 
 def import_diagnostics_data(event, context):  # NOSONAR
     file_name = event.get('name')
-    logging.info("Processing file: %s." % file_name)
-    data = download_file(file_name)
-    logging.info("Inserting file %s into BigQuery." % file_name)
-    insert_into_bigquery(data)
-    logging.info("Deleting file: %s." % file_name)
-    delete_file(file_name)
-    logging.info("Finished importing file: %s." % file_name)
+    logging.info(f"Processing file: {file_name}.")
```

Comment:
Good to change this too. But we don't currently have any automated deployment to Cloud Functions. @htsili how do you recommend we handle this? Should we just give @pritamghanghas write access to CF, have you do it, or automate things?

Reply:
@marvinmarnold 
there are several frameworks but most of them target AWS. Very few support gcp. Serverless framework does support gcp fit seems.